keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/InferAndroidTestKeepRules.kt

```diff
@@ -92,9 +94,25 @@ abstract class InferAndroidTestKeepRules : JavaExec() {
         androidTestJarProvider: TaskProvider<out Jar>,
         releaseClassesJarProvider: TaskProvider<out Jar>,
         androidJar: Provider<RegularFile>,
+        automaticallyAddR8Repo: Property<Boolean>,
         extensionJvmArgs: ListProperty<String>,
         r8Configuration: Configuration
     ): InferAndroidTestKeepRules.() -> Unit = {
+      if (automaticallyAddR8Repo.get()) {
```

Comment:
Did you move this from the plugin to the task for any particular reason?

Reply:
Yeah had to to move it to configuration time